data_type: "pcd"
stop_each_frame: true
pub_ground_truth: false
pub_frequency: 10.0
data_path: "/home/awesome/Documents/Datasets/DOALS/00"
start_frame: 0      # kitti: 00:4390 01:150 02:860 05:2350 07:630, doals: 0
end_frame: 10000        # kitti: 00:4530 01:250 02:950 05:2670 07:820, doals: 250
min_range: 0.0
max_range: 80.0
delta_range: 0.25
delta_azimuth: 0.5
delta_polar: 2.0
res_range: 0.1
res_azimuth: 0.25
downsample: true
voxel_leaf: 0.2
window_size: 10
map_size: 2           # kitti: 00:5 
occ: 0.9              # kitti: 00:3.0 
map_gap: 3
scan_to_map: true